### PR TITLE
[TRAFODION-1740] Add CQDs to UPDATE STATS for large tables

### PR DESCRIPTION
**Pull request by DaveBirdsall**

### CODE REVIEW

**Comment by zellerh** [comment]
Looks good to me!

File: core/sql/ustat/hs_globals.cpp [hunk 1]
```diff
@@ -3762,6 +3765,68 @@ Lng32 HSSample::make(NABoolean rowCountIsEstimate, // input
...
+              workableCacheSize = 1;  // can't go below 1 unfortunately
+
+            Int32 max = getDefaultAsLong(HBASE_NUM_CACHE_ROWS_MAX);
+            if ((workableCacheSize < 10000) && // don't bother if 10000 works
+                (max == 10000))  // don't do it if user has already set this CQD
+              {
+                char temp1[40];  // way more space than needed, but it's safe
+                Lng32 wcs = (Lng32)workableCacheSize;  
+                sprintf(temp1,"'%d'",wcs);
+                NAString minCQD = "CONTROL QUERY HBASE_NUM_CACHE_ROWS_MIN ";
```
Comment (DaveBirdsall) [inline core/sql/ustat/hs_globals.cpp]: Bug: Need to add the DEFAULT keyword after CONTROL QUERY

File: core/sql/ustat/hs_globals.cpp [hunk 2]
```diff
@@ -3762,6 +3765,68 @@ Lng32 HSSample::make(NABoolean rowCountIsEstimate, // input
...
+                minCQD += temp1;
+                HSFuncExecQuery(minCQD); 
+                NAString maxCQD = "CONTROL QUERY HBASE_NUM_CACHE_ROWS_MAX ";
```
Comment (DaveBirdsall) [inline core/sql/ustat/hs_globals.cpp]: Same here.

**Comment by DaveBirdsall** [comment]
Don't merge this yet. Found a couple of bugs. (Good thing I continued unit testing after submitting the pull request.)

**Pull request merged**